src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -10,6 +10,7 @@
     <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)' != 'Current'">$(MSBuildAllProjects);$(MsBuildThisFileFullPath)</MSBuildAllProjects>
     <SqlServerVersion>Sql150</SqlServerVersion>
     <TargetExt>.dacpac</TargetExt>
+    <PackageType>Dacpac</PackageType>
```

Comment:
I'm not sure we can actually do this. This might make the resulting package uninstallable into another project.

Reply:
We do it today in the .nuspec - are you saying it is not important?

Reply:
Do we? I vaguely remember having issues with that before, but maybe something changed in VS. Perhaps we should test to make sure that the package can still be installed into a project. If that works then I'm fine with this change.

Reply:
The package type is there today, yes. Not sure what you mean by: "that the package can still be installed into a project." - that the package can be consumed by another MsBuild.Sdk:SqlProj project?

Reply:
Yes, that is what I mean :).

Reply:
OK, I will test that!

Reply:
It looks like we already test that in the pipeline: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/test/TestProjectWithSDKRef/TestProjectWithSDKRef.csproj#L12

Reply:
I'm not entirely sure that that is the same as doing it from VS though. From what I remember it was a VS limitation at the time.

Reply:
Ah, I will test in VS with a local NuGet feed

Reply:
Thanks, I was able to repro!